release-notes/opensearch-dashboards.release-notes-2.15.0.md

```diff
@@ -1,78 +1,78 @@
 ## What's Changed
 ### 📈 Features/Enhancements
```

Comment:
shall we also manually clean out changelogs/fragments to remove the two PRs 7054.yml, 6320.yml

Reply:
No. IMO changelog and fragments should be in sync and retain everything done, including rollbacks. However, release notes needn't have rollbacks within the same release.